src/components/Ruler/index.js

```diff
@@ -40,12 +40,14 @@ const calculateY = (points, yTooltipPosition) => {
 };
 
 const Ruler = ({ ruler, points, height, width }) => {
-  const xTooltipMargin = 5;
+  const timeLabelMargin = 5;
   // The fixed position of a x-axis label which is the same for all highlighted points
-  const xTooltipPosition = height - labelHeight - xTooltipMargin;
-  const pointsObject = calculateY(points, xTooltipPosition);
+  const timeTooltipPosition =
+    typeof ruler.getTimeLabelPosition === 'function'
+      ? ruler.getTimeLabelPosition(height, labelHeight)
```

Comment:
Thoughts on passing in `height - labelHeight - timeLabelMargin`? (Basically "if you don't do anything, this is where you'll be")

Reply:
I think that could work too. Which one do you think would be better?

Reply:
I just sorta see these params as ".... what am I supposed to do with these? What does the library do with these?"

I think it would be more self-documenting if it got something like `getTimeLabelPosition(position, { height, labelHeight, xTooltipMargin })` so I (the outside app) could see the result, and its component parts.